Airway/Breathing/Circulation/Pharma/Zeus - Add - German Translation Update

OlafZigel opened the pull request:
**When merged this pull request will:**
- _added missing entries for translation_


### IMPORTANT

- [Development Guidelines](https://ace3.acemod.org/wiki/development/) are read, understood and applied.
- Title of this PR uses our standard template `Component - Add|Fix|Improve|Change|Make|Remove {changes}`.

OlafZigel committed:
Deutsch 05.03.2024

MissHeda commented on addons/airway/stringtable.xml:
```diff
@@ -688,35 +698,42 @@
         <Key ID="STR_KAT_Airway_Suction_Log">
             <English>%1 did a medical suction using a %2</English>
             <Japanese>%1 は %2 で医療吸引を行った</Japanese>
+            <German>%1 hat eine medizinische Absaugung durchgeführt mit einem %2</German>
```
Ändere diesen string das es sinn ergibt wenn das Item (%2) eine "Manuelle Absaugpumpe" ist. %1 ist hier der spieler / medic der das absaugen macht.

OlafZigel replied:
Also wäre es logischer, wenn man folgendes schreiben würde? 

%1 hat eine medizinische Absaugung durchgeführt mit einer %2

einem -> einer

MissHeda replied:
Dann würde der Accuvac nicht mehr rein passen.

OverlordZorn replied:
%1 hat eine medizinische Absaugung mittels %2 durchgeführt

MissHeda commented on addons/airway/stringtable.xml:
```diff
@@ -884,11 +901,13 @@
             <English>Training level required to use an Manual Suction Pump</English>
             <Japanese>手動吸引器の使用に必要な医療スキルのレベルを設定します。</Japanese>
             <Finnish>Aika tarkistaa hengitystiet</Finnish>
+            <German>Für die Verwendung einer manuellen Absaugpumpe ist ein Schulungsniveau erforderlich</German>
```
Prüfe andere strings medlvl

MissHeda commented on addons/airway/stringtable.xml:
```diff
@@ -1035,11 +1054,13 @@
             <English>Time for Manual Suction Pump</English>
             <Japanese>手動吸引器の所要時間</Japanese>
             <Finnish>aika manuaaliselle imupumpulle</Finnish>
+            <German>Zeit für eine manuelle Absaugpumpe</German>
```
Prüfe andere strings treatmentTime

MissHeda commented on addons/airway/stringtable.xml:
```diff
@@ -1035,11 +1054,13 @@
             <English>How long it will take to use Manual Suction Pump</English>
             <Japanese>手動吸引器を使用するのにどのくらい時間がかかるか設定します</Japanese>
             <Finnish>Kuinka kauan manuaalisen imupumpun käyttö kestää</Finnish>
+            <German>Wie lange dauert die Verwendung der manuellen Absaugpumpe?</German>
```
Prüfe andere strings treatmentTime

MissHeda commented on addons/airway/stringtable.xml:
```diff
@@ -1303,6 +1325,7 @@
             <Norwegian>Hvor lang tid det vil ta å manuelt kansellere stabilt sideleie</Norwegian>
             <French>Combien de temps sera nécessaire pour annuler la position latérale de sécurité</French>
             <Finnish>Kuinka kauan lepoasennon manuaalinen peruuttaminen kestää</Finnish>
+            <German>Wie lange dauert es, die stabile Seitenlage manuell abzubrechen</German>
```
Prüfe andere strings treatmentTime

MissHeda commented on addons/airway/stringtable.xml:
```diff
@@ -1311,6 +1334,7 @@
             <Norwegian>Tid i stabilt sideleie for å bli kvitt okklusjonen</Norwegian>
             <French>Temps pour retirer l'occlusion en position latérale de sécurité</French>
             <Finnish>Palautuminen Asento-aika tyhjennystukoksen poistamiseksi</Finnish>
+            <German>Stabile Seitenlage um die verstopfung zu entfernen</German>
```
Orentiere dich an anderen strings wie Occlusion übersetzt wurde

MissHeda commented on addons/airway/stringtable.xml:
```diff
@@ -1319,6 +1343,7 @@
             <Norwegian>Maks nødvendig tid en pasient må være i stabilt sideleie for å bli kvitt okklusjonen.</Norwegian>
             <French>Temps maximal pour qu'une occlusion disparaisse sur un patient en position latérale de sécurité</French>
             <Finnish>Enimmäisaika, joka tarvitaan potilaan puhdistamiseen toipumisasennossa</Finnish>
+            <German>Maximale Zeit, die der Patient in der stabilen Seitenlage benötigt, um die verstopfung zu beseitigen.</German>
```
Orentiere dich an anderen strings wie Occlusion übersetzt wurde

MissHeda commented on addons/breathing/stringtable.xml:
```diff
@@ -1424,6 +1425,7 @@
             <Japanese>呼吸補助を開始した</Japanese>
             <French>Début des insufflations</French>
             <Finnish>Pelastushengitykset alkavat</Finnish>
+            <German>Atemspende begonnen</German>
```
Benutzt du google translate?
```suggestion
            <German>Künstliche Beatmung Begonnen</German>
```

MissHeda commented on addons/circulation/stringtable.xml:
```diff
@@ -20,7 +20,7 @@
         <Key ID="STR_KAT_Circulation_SETTING_AdvRhythm_Enable">
             <English>Advanced Rhythms activated?</English>
-            <German>Fortgeschrittenen Rhythmus aktivieren?</German>
+            <German>Fortgeschrittenen Rhythmus aktiviert?</German>
```
streichen

MissHeda commented on addons/circulation/stringtable.xml:
```diff
@@ -1727,6 +1729,7 @@
         <Key ID="STR_KAT_Circulation_bloodtype_log">
             <English>%1 checked dogtags blood group: %2</English>
             <Japanese>%1 が認識票の血液型を確認: %2</Japanese>
+            <German>%1 hat die Erkennungsmarken Blutgruppe überprüft: %2</German>
```
```suggestion
            <German>%1 hat die Blutgruppe der Erkennungsmarke überprüft: %2</German>
```

MissHeda commented on addons/circulation/stringtable.xml:
```diff
@@ -2009,6 +2012,7 @@
         <Key ID="STR_KAT_Circulation_SETTING_AEDStation_AllowInteraction_DESCRIPTION">
             <English>Medical level required to pick up placed AED/X station</English>
             <Japanese>設置されたAED/Xを拾うのに必要な医療スキルのレベルを設定します。</Japanese>
+            <German>Für die Aufnahme der platzierten AED/X-Station ist ein medizinisches Niveau erforderlich</German>
```
Prüfe andere strings medlvl

MissHeda commented on addons/pharma/stringtable.xml:
```diff
@@ -1105,9 +1105,11 @@
         </Key>
         <Key ID="STR_KAT_Pharma_SETTING_REQUIRE_INS_IV_BLOOD_DRAW_DESC">
             <English>Default false. If true, you will need to insert IV/IO first before being able to draw blood from someone. \nIf false, you can draw without it.</English>
+            <German>Standardmäßig falsch. If true,  müssen Sie zuerst eine Infusion/Infusion einführen, bevor Sie jemandem Blut abnehmen können. \nIf false, können Sie ohne zeichnen.</German>
```
```suggestion
            <English>Default false. If true, you will need to insert IV first before being able to draw blood from someone. \nIf false, you can draw without it.</English>
            <German>Standardmäßig falsch. Bei "wahr" muss zuerst ein IV gelegt sein, bevor jemandem Blut entnehmen kann. \nBei "falsch" kann Blut ohne IV entnommen.</German>
```

MissHeda commented on addons/pharma/stringtable.xml:
```diff
@@ -1105,9 +1105,11 @@
         </Key>
         <Key ID="STR_KAT_Pharma_SETTING_REQUIRE_INS_IV_BLOOD_DRAW">
             <English>Blood Drawing Require IV/IO</English>
+            <German>Blutentnahme erfordert IV/IO</German>
```
Das IO wurde vergessen raus zu nehmen wäre nett wenn du das gleich so mit ändern könntest.
```suggestion
            <English>Blood Drawing Require IV</English>
            <German>Blutentnahme erfordert IV</German>
```

OlafZigel committed:
Updated Translation

OlafZigel committed:
Updated Translation V3

MiszczuZPolski committed:
fix indent

MiszczuZPolski committed:
fix duplicate

OlafZigel committed:
Update Translation V4

fixed indent [addons/pharma/stringtable.xml] line: 1111

deleted duplicate [addons/airway/stringtable.xml] line: 1330

OlafZigel committed:
Merge branch 'Translation-German' of https://github.com/OlafZigel/KAM into Translation-German

OverlordZorn commented on addons/airway/stringtable.xml:
```diff
@@ -319,7 +319,7 @@
             <French>Voies aériennes obstruées, geste médical nécessaire</French>
             <Finnish>Hengitystiet ovat tukossa, hengityshoitoa tarvitaan</Finnish>
-            <German>Die Atemwege sind blockiert, eine Atemwegskontrolle ist erforderlich</German>
+            <German>Die Atemwege sind blockiert, Atemwegssicherung ist erforderlich</German>
```
Is Atemwegssicherung ein gängeriger Begriff im San Dienst? Atemswegskontrolle scheint mir zumindest viel geläufiger.

OlafZigel replied:
Die aktion ist bereits die atemwegskontrolle

OverlordZorn replied:
Alternativ wäre dann noch ` ..., die Atemwege müssen freigelegt werden`

OverlordZorn commented on addons/airway/stringtable.xml:
```diff
@@ -1326,8 +1326,6 @@
             <French>Combien de temps sera nécessaire pour annuler la position latérale de sécurité</French>
             <Finnish>Kuinka kauan lepoasennon manuaalinen peruuttaminen kestää</Finnish>
             <German>Wie lang die benötigte Zeit ist, die stabile Seitenlage manuell abzubrechen</German>
```
Benötigte Zeit, um stabile Seitenlage manuell abzubrechen.

OverlordZorn commented on addons/airway/stringtable.xml:
```diff
@@ -70,7 +70,7 @@
         <Key ID="STR_KAT_Airway_SETTING_occluded_DESC">
             <English>Probability for an occlusion in percentage that gets rolled Setting:[Occlusion repeat timer] (default every 60s)</English>
-            <German>Atemwegsverlegungswahrscheinlichkeit durch Erbrochenes in prozent die durch Einstellung:[Zeit für die Wiederholung des Erbrechens] (standard alle 60s) berechnet wird.</German>
+            <German>Wahrscheinlichkeit für eine Okklusion in Prozent, die gewürfelt wird. Einstellung: [Timer für Okklusionswiederholung] (Standard alle 60 Sekunden)</German>
```
> Wahrscheinlichkeit für eine Okklusion in Prozent, die gewürfelt wird. 
`Wahrscheinlichkeit für eine Okklusion in Prozent.` Sollte ausreichend sein, der Nebensatz ist nicht wirklich nötig.

@MiszczuZPolski the english orginial is a bit wierd, ngl. Whats the context here?

OverlordZorn commented on addons/airway/stringtable.xml:
```diff
@@ -83,7 +83,7 @@
         <Key ID="STR_KAT_Airway_SETTING_occlusion_repeatTimer">
             <English>Occlusion repeat timer</English>
-            <German>Zeit für die Wiederholung des Erbrechens</German>
+            <German>Okklusionswiederholungstimer</German>
```
`Timer für den Wiedereintritt einer Okklusion`
`Zeit zum Wiedereintritt einer Okklusion`
Würde sich besser anhören. 
@MissHeda kenn den Kontext hier nicht 100%, wäre das so sinnig?

OverlordZorn replied:
Alternativ: Zeitfenster zum Wiedereintritt einer Okklusion

MissHeda replied:
Würde ich so unterschreiben

OlafZigel committed:
Updated Translation V5

OlafZigel committed:
Updated Translation V6

OlafZigel committed:
Updated Translation V7

OlafZigel committed:
Merge branch 'dev-Tomcat' into Translation-German

OlafZigel committed:
Updated Translation V8

OlafZigel committed:
Updated Translation V9

Idk what why it has changed the Japanese Translation. I hope it is still right :smiley:

OlafZigel committed:
Merge branch 'dev-Tomcat' into Translation-German

OlafZigel committed:
Updated Translation V10

I hope it is the last last one and the japanese is correct. If not, fix it by urself

OlafZigel committed:
Updated Translation V11

LAST ONE!

OlafZigel commented:
It's done and merged into #506
